modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -278,13 +304,26 @@ export default class NestedTabs extends Base {
 		if ( this.getSettings( 'autoExpand' ) ) {
 			this.activateDefaultTab();
 		}
+
+		this.setHorizontalScrollAlignment();
 	}
 
 	onEditSettingsChange( propertyName, value ) {
 		if ( 'activeItemIndex' === propertyName ) {
 			this.changeActiveTab( value, false );
 		}
 	}
+	onElementChange( propertyName ) {
+		if ( this.checkSliderPropsToWatch( propertyName ) ) {
+			this.setHorizontalScrollAlignment();
+		}
+	}
+
+	checkSliderPropsToWatch( propertyName ) {
+		return 0 === propertyName.indexOf( 'horizontal_scroll' ) ||
+			0 === propertyName.indexOf( 'tabs_justify_horizontal' ) ||
+			0 === propertyName.indexOf( 'tabs_title_space_between' );
+	}
```

Comment:
I am not sure if I understand what happens here.
I assume we only use the horizontal scroll function when the `propertyName` horizontal_scroll` has been activated.

Why do we check `tabs_justify_horizontal` and `tabs_title_space_between` as well?

About `propertyName`: I would assume that always to return a string, but you seem to treat it like an array. Are you sure about that? I am just curious.

Reply:
@ArmanPurtoian @louiswol94 
100%